src/components/ChatApp.css

```diff
@@ -6,26 +6,73 @@
   overflow: hidden;
 }
 
-.message-list, .thread-list {
-  border: 1px solid #38c3ea;
-  font-size: 16px;
-  height: 400px;
-  margin: 0;
+.message-list {
+  background: #eceff1;
+  position: fixed;
+  top: 56px;
+  left: 0;
```

Comment:
We have `margin: 0 auto` to place it in middle, still using `left:0` and `right:0` ?